components/CollectorPkgInstallInstructions.tsx

```diff
@@ -54,19 +54,21 @@ const CollectorDistroInstallInstructions: React.FunctionComponent<Pick<Props, 'k
   switch (kind) {
     case 'yum':
       installOpts = [
+        [ 'el9', 'el/9', 'RHEL / Rocky / OL 9' ],
```

Comment:
Interestingly, this is the only place we mention "Oracle Linux". Is it really necessary? I don't think we even test with it really (technically we don't test with Rocky with EL9 too).

Reply:
We did add it intentionally: if you're running OL9 I guess you may not know it's a RHEL derivative? I agree it makes this busier, but that clarity might be worth it.